edx_proctoring/api.py

```diff
@@ -1526,6 +1526,23 @@ def _calculate_allowed_mins(due_datetime, allowed_mins):
     return actual_allowed_mins, is_exam_past_due_date
 
 
+def _is_self_paced_course(user_id, course_id):
```

Comment:
@schenedx But what if the due date sent in the `context` object has passed? We already were checking this due date with `has_due_date_passed(exam['due_date'])` at line 1427 of this file. Are you working under the assumption that setting the self-paced flag on the course will overwrite the `due` field on the `CourseModule`? I'm guessing that does not happen, otherwise this would already work as expected, no?